static/js/reducers/index.js

```diff
@@ -68,7 +68,16 @@ export const profiles = (state: Profiles = INITIAL_PROFILES_STATE, action: Actio
       clone[username],
       newProfile
     );
-    return clone;
+    return {
+      ...state,
+      [username]: {
+        ...{
+          profile: {}
+        },
+        ...state[username],
+        ...newProfile,
+      },
+    };
```

Comment:
it looks here like the `clone` object declared above isn't used, can we delete those lines?

also what would you think about adjusting the formatting a little to look something like this:

```js
return {
  ...state,
  [username]: {
    ...{ profile: {} },
    ...state[username],
    ...newProfile,
  },
};
```